src/Appwrite/Utopia/Database/Validator/Queries/Base.php

```diff
@@ -22,7 +23,7 @@ class Base extends Queries
      * @param string[] $allowedAttributes
      * @throws \Exception
      */
-    public function __construct(string $collection, array $allowedAttributes)
+    public function __construct(string $collection, array $allowedAttributes, ?array $prohibitedQueries = [])
```

Comment:
Since we want to iterate the array and it has a default of empty, let's just use `array` as the type so that `null` can't be passed

Reply:
done this